code/game/machinery/_machinery.dm

```diff
@@ -114,6 +114,14 @@ Class Procs:
 	var/speed_process = FALSE // Process as fast as possible?
 	var/obj/item/circuitboard/circuit // Circuit to be created and inserted when the machinery is created
 
+	var/obj/item/card/id/inserted_scan_id
+	var/obj/item/card/id/inserted_modify_id
+	var/list/region_access = null // For the identification console (card.dm)
+	var/list/head_subordinates = null // For the identification console (card.dm)
+	var/authenticated = 0 // For the identification console (card.dm)
```

Comment:
Why are these duplicated (here and computer/card.dm)

Reply:
This has nothing to do with plumbing..?

Reply:
blame https://github.com/Citadel-Station-13/Citadel-Station-13/pull/9829 and https://github.com/Citadel-Station-13/Citadel-Station-13/pull/9826